account_move_budget/views/res_partner_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form_inherit" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button
+                    class="oe_stat_button"
```

Comment:
Perhaps hide if there are no budgets? Just as a suggestion, maybe it is relevant to also have it displayed for all Contacts in your use case. 🤔

Reply:
Yeah I wasn't sure for what option to go for. Having it shown when it is also 0 is a quick way to know if the partner is included in any budget. There is no other quick way to know it.

Reply:
Sure, makes sense 👍🏿